CLI.md

```diff
@@ -56,7 +56,7 @@ A Byzantine-fault tolerant sidechain with low-latency finality and high throughp
 * `subscribe` — Subscribes to a system channel
 * `unsubscribe` — Unsubscribes from a system channel
 * `open-multi-owner-chain` — Open (i.e. activate) a new multi-owner chain deriving the UID from an existing one
-* `close-chain` — Close (i.e. deactivate) an existing chain
+* `close-chain` — Close an existing chain
```

Comment:
Need to decide for or against the `-s`

Reply:
The `-s` is just a Rust convention. On the command line it seems we use the imperative.

Reply:
Sg. Can we fix "(un)subscribes" then?